src/client/pages/OfferNew/useQuoteIds.ts

```diff
@@ -0,0 +1,52 @@
+import queryString from 'querystring'
+import { Location } from 'history'
+import { useEffect, useState } from 'react'
+import { useLocation } from 'react-router'
+import { StorageState, useStorage } from '../../utils/StorageContainer'
+
+/**
+ * Hook that allows general access to quote IDs.
+ * @returns Either an array of quote IDs, an empty array if none were found,
+ *          or `null` before quote IDs have been loaded.
+ */
```

Comment:
I'm not sure what I think about comments like these. 🤔 But I'm leaning towards that the code is clear enough so that it's not needed - it was just me being inattentive(sloppy? 😳 ) looking at the code before I think.